packages/core/src/html/CustomListIconProcessor.ts

```diff
@@ -25,9 +25,40 @@ interface IconAttributes {
   once?: boolean;
 }
 
+class TextsManager {
+  texts: string[] = [];
+  pt: number = 0;
+  size: number = 0;
+  constructor() {
+    this.texts = [];
+  }
+
+  isUsed() {
+    return this.size > 0;
+  }
+
+  next(): string {
+    if (this.size === 0) {
+      throw new Error('No texts');
+    }
+    const next = this.texts[this.pt];
+    if (this.pt <= this.size - 1) {
+      this.pt += 1;
+    }
+    return next;
+  }
+
+  resetTexts(texts: string[]) {
+    this.texts = texts.map(text => text.trim());
+    this.size = texts.length;
+    this.pt = 0;
+  }
+}
+
 type IconAttributeDetail = {
   isFirst: boolean;
   addIcons: boolean;
+  texts: TextsManager;
```

Comment:
This naming is misleading: its `texts` but the actual object is a `textsManager`

Reply:
Thank you for this suggestion. Modified as suggested in later commit.